package.json

```diff
@@ -1,25 +1,28 @@
 {
```

Comment:
This file should actually use 2-space indentation, as it's essentially a "binary" file written by things like `npm install --save"